src/migration-scripts/bgp/2-to-3

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2022 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# T4257: Discussion on changing BGP autonomous system number syntax
+
+from sys import argv
+from sys import exit
+
+from vyos.configtree import ConfigTree
+
+if (len(argv) < 1):
+    print("Must specify file name!")
+    exit(1)
+
+file_name = argv[1]
+
+with open(file_name, 'r') as f:
+    config_file = f.read()
+
+config = ConfigTree(config_file)
+
+# Check if local-as exists, then add the system-as, then remove the local-as. This is for global configuration.
+if config.exists(['protocols', 'bgp', 'local-as']):
+    config.set(['protocols', 'bgp', 'system-as'], value = config.return_value(['protocols', 'bgp', 'local-as']))
+    config.delete(['protocols', 'bgp', 'local-as'])
+
+# Check if local-as exists inside of vrf, then add the system-as, then remove the local-as. This is for vrf configuration.
+if config.exists(['vrf']):
+    for vrf in config.list_nodes(['vrf', 'name']):
```

Comment:
A think there should be an additional check as we can have `vrf` option without `name`
For example:
```
set vrf bind-to-all
```

Reply:
Check and see if the changes I made here are good. I *think* this should exit out if vrf name is not present...but only then. It shouldn't match on the other option.